user-access-service/src/test/java/no/unit/nva/database/IdentityServiceImplTest.java

```diff
@@ -51,6 +56,24 @@ public void getRoleLogsWarningWhenNotFoundExceptionIsThrown() throws InvalidEntr
                    StringContains.containsString(ROLE_NOT_FOUND_MESSAGE));
     }
 
+    @Test
+    void shouldSucceedUpdatingAnExistingRole() throws InvalidInputException, ConflictException {
+        var serviceUnderTest = new IdentityServiceImpl(initializeTestDatabase());
```

Comment:
the name `serviceUnderTest `doens't give me any clue as to why `databaseService  `cannot be used
consider renaming serviceUnderTest to something that hints as to why you need another instance of IdentiyService

Reply:
Actually, it could be `databaseService`. Fixed.